app/frontend/schools/settings/types/SchoolCustomize__Customizations.res

```diff
@@ -83,14 +88,41 @@ let removeLink = (linkId, t) => {
   ...t,
   links: t.links |> List.filter(l =>
     switch l {
-    | HeaderLink(id, _, _)
-    | FooterLink(id, _, _) =>
+    | HeaderLink(id, _, _, _)
+    | FooterLink(id, _, _, _) =>
       id != linkId
-    | SocialLink(id, _) => id != linkId
+    | SocialLink(id, _, _) => id != linkId
     }
   ),
 }
 
+let updateLink = (linkId, newTitle, newUrl, t) => {
+  ...t,
+  links: t.links |> List.map(l =>
```

Comment:
We are switching away from using `List` to `Js.Array`. I think we should use `Js.Array2.map` here.